geode-core/src/test/java/org/apache/geode/internal/cache/partitioned/rebalance/PartitionedRegionLoadModelJUnitTest.java

```diff
@@ -483,30 +485,73 @@ public void testColocationEnforceLocalMaxMemory() throws UnknownHostException {
   }
 
   /**
-   * Test that each region individually honors it's enforce local max memory flag.
+   * Test that a region with enforceLocalMaxMemory disabled colocated with
+   * a region that memory full with enforceLocalMaxMemory enabled will prevent a bucket move.
```

Comment:
I would change this sentence to: a region with memory full and enforceLocalmaxMemory enabled...